app/models/food_item.rb

```diff
@@ -0,0 +1,20 @@
+class FoodItem < ApplicationRecord
+  has_many :meals
+  has_many :menus, through: :meals
+
+  def self.all_meals
```

Comment:
first of all
`FoodItem.pluck(:meal_type).uniq`

next - use enum instead

[source](http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html)